Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'auto_strip_attributes', '~> 2.0', '>= 2.0.6'
 gem 'httparty', '~> 0.13.7'
 gem 'addressable', '~> 2.3.7'
 gem 'typhoeus', '~> 1.0', '>= 1.0.1'
+gem 'rinku', '~> 1.7', '>= 1.7.3', require: "rails_rinku"
```

Comment:
This particular gem only patches rails `auto_link` method if you require `rails_rinku`. It literally has no other use to us, so I added it in here.